src/amber/support/dsls.cr

```diff
@@ -23,4 +23,24 @@ module Amber::Support::DSL
 
     {% end %}
   end
+
+  module ControllerActions
+    macro render_both(filename, layout)
```

Comment:
To explicitly render templates and layouts in any location. I also call it from render to keep the functionality dry.